src/main/java/dev/kurumiDisciples/chisataki/commands/slash/IgnoreCommand.java

```diff
@@ -20,58 +34,113 @@ public IgnoreCommand() {
+			statement.setLong(1, guildId);
+			statement.setLong(2, memberId);
+
+			ResultSet set = statement.executeQuery();
+			return set.next();
+		}
+		catch (SQLException | InitializationException e){
+			LOGGER.error("An error occured in IgnoreCommand when retriving table", e);
+			return false;
 		}
-		return false;
 	}
 
-	private static JsonArray removeMember(String memberId) {
-		JsonArrayBuilder jsonArray = Json.createArrayBuilder();
-		JsonArray list = getIgnoreList();
-		
-		for (int i = 0; i < list.size(); i++){
-			if (!list.getString(i).equals(memberId))
-				jsonArray.add(list.getString(i));
+	public static boolean isMemberIgnored(String memberId){
+		/* Assume ChisaTaki server */
+		return isMemberIgnored(1010078628761055234L, Long.parseLong(memberId));
+	}
+	private static void removeMember(long guildId, long memberId) {
+		try(PreparedStatement statement = Database.createStatement(REMOVE_IGNORED_USER)){
+			statement.setLong(1, guildId);
+			statement.setLong(2, memberId);
+			statement.executeUpdate();
+			LOGGER.info("Removed member {} from ignore list", memberId);
+		}
+		catch (SQLException | InitializationException e){
+			LOGGER.error("An error occured in IgnoreCommand when removing from table", e);
 		}
-		
-		return jsonArray.build();
 	}
 	
-	private static JsonArray insertMember(String memberId) {
-		JsonArrayBuilder jsonArray = Json.createArrayBuilder();
-		JsonArray previousList = getIgnoreList();
-		
-		for (JsonValue value : previousList){
-			jsonArray.add(value);
+	private static void insertMember(long guildId, long memberId) {
+		try(PreparedStatement statement = Database.createStatement(INSERT_IGNORED_USERS)){
+			statement.setLong(1, guildId);
+			statement.setLong(2, memberId);
+			statement.executeUpdate();
+		}
+		catch (SQLException | InitializationException e){
+			LOGGER.error("An error occured in IgnoreCommand when inserting into table", e);
 		}
-		jsonArray.add(memberId);
-		
-		return jsonArray.build();
 	}
 
-	private static JsonArray getIgnoreList() {
-		return FileUtils.getFileContentArray(FILE_PATH, "ignore");
+	private static List<String> getIgnoreList() {
+		List<String> list = new ArrayList<>();
+		try(PreparedStatement statement = Database.createStatement(GET_ALL_IGNORED_USERS)){
+			/* Assume ChisaTaki server */
+			statement.setLong(1, 1010078628761055234L);
+			ResultSet set = statement.executeQuery();
+			while (set.next()){
+				list.add(set.getString("member_id"));
+			}
+			return list;
+		}
+		catch (SQLException | InitializationException e){
+			LOGGER.error("An error occured in IgnoreCommand when retriving table", e);
+			return null;
+		}
+	}
+
+	@Override
+	public String getPrimaryKey(){
+		return null;
+	}
+
+	@Override
+	public String getTableName(){
+		return "ignore_list";
+	}
+
+	@Override
+	public HashMap<String, Integer> getDefinedColumns(){
+		HashMap<String, Integer> columns = new HashMap<>();
+		columns.put("guild_id", Types.VARCHAR);
```

Comment:
Is there really a need for a guild column when other tables like cooldown and shrines don't use it + this bot is for cstk only?